README.md

```diff
@@ -29,7 +29,14 @@ This is a resource for anyone visiting Denver or people recently new to the Denv
 * [#topic-gigs](https://denver-devs.slack.com/messages/topic-gigs/) in the [Denver Devs Slack](https://denver-dev-slack.herokuapp.com/)
 
 ## Boulder & Denver
-What's the diff? How is the commute?
+From the perspective of someone who migrated to Colorado and has happily lived in both towns for 3 years each:
+### Denver
+Denver is a mid-sized American city. It is large enough to have a team in every major sport (and some non-major ones), an Arts center, and an increasingly diverse population. Through intelligent planning, it is also increasingly bike/walk friendly and has decent public transportation infrastructure. Crime is relatively low. It avoids the worst traffic issues of major American cities, while still having plenty (25 South in the mornings, anyone?). The economy is growing right now, so jobs are fairly plentiful, and the Mayor and Governor (who is a former Mayor) have pushed for tech investment. That said,
+### Boulder
+Boulder is an elite college town. Maybe more accurately, it is an elite town, and a college town. The best description I have personally heard is "Wow, it's like Berkeley, without all the dirty parts." It is an interesting city because of its conflicting natures. The best aspects of Boulder are that it is a pretty chill place to live. You can bike everywhere, safely and comfortably, in some of the most beautiful surroundings in the country. Seriously. They snow-plow the bike paths before they plow the roads. There are busses that go everywhere with cute names like the Skip, Hop and Dash! There is the culture and stuff-to-do that comes from having an R1 university in the town. You are literally right next to the mountains, as in, you get to the edge of town and you are climbing. It's an almost perfect blend of nature, city and small town. The recreation centers are excellent and affordable, but you won't need them because you'll just be outside all the time.
```

Comment:
See if you could rework that "The best description I have personally heard is "Wow, it's like Berkeley, without all the dirty parts." - to change the language from "I" - maybe "Some say "Wow, it's like Berkeley, without all the dirty parts"